save-common/src/commonMain/kotlin/org/cqfn/save/entities/Project.kt

```diff
@@ -1,19 +1,54 @@
 package org.cqfn.save.entities
 
+import kotlinx.serialization.Serializable
+
 /**
  * @property owner
  * @property name
  * @property type type of the project, e.g. github or manually registered
  * @property url
  * @property description description of the project, may be absent
- * @property id
  */
 @Entity
-class Project(
+@Serializable
+data class Project(
```

Comment:
`@Entity` shouldn't be a data class

Reply:
? https://github.com/cqfn/save-cloud/pull/50/files/9f04ecdab44b901f02cbbe909f034cf101a7b99b..d6f889d2825b42a899deb19d61bf6a84ce146c39#r597583407